cli/actions/__tests__/lint.test.js

```diff
@@ -0,0 +1,39 @@
+jest.setMock('glob', {
+  sync: () => ({ length: 1 }),
+});
+jest.mock('../../logger');
+jest.mock('../../../utils/paths');
+
+describe('lint', () => {
+  global.process.exit = jest.fn();
+  const logger = require('../../logger');
+  const lint = require('../lint');
+  const eslint = require('eslint');
+
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  it('logs and exits 0 when no errors', () => {
+    lint();
+    expect(logger.end).toBeCalledWith('Your JS looks great ✨');
```

Comment:
can we test partial strings with this? seems brittle to include emojis and everything